mlflow/tracking/fluent.py

```diff
@@ -369,7 +609,32 @@ def delete_experiment(experiment_id):
     """
     Delete an experiment from the backend store.
 
-    :param experiment_id: The experiment ID returned from ``create_experiment``.
+    :param experiment_id: The The string-ified experiment ID returned from ``create_experiment``.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Convert experiment ID as a string argument
+        mlflow.delete_experiment(str(1))
+
+        # Examine the deleted experiment details.
+        # Note: Deleted experiments are moved to a .thrash folder under
+        # the artifact location top level directory.
+        experiment = mlflow.get_experiment(str(1))
```

Comment:
```suggestion
        experiment = mlflow.get_experiment(str("1"))
```

Reply:
This can be changed to something like this a user can run:

```
experiment_id = mlflow.create_experiment("New Experiment")
mlflow.delete_experiment(experiment_id)

 # Examine the deleted experiment details. Deleted experiments
 # are moved to a .thrash folder under the artifact location top
 # level directory.
 experiment = mlflow.get_experiment(experiment_id)

 # Print the contents of deleted Experiment data
 print("Name: {}".format(experiment.name))
 print("Artifact Location: {}".format(experiment.artifact_location))
 print("Tags: {}".format(experiment.tags))
 print("Lifecycle_stage: {}".format(experiment.lifecycle_stage))
```